### PR TITLE
Use new common benchmark config

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/CoreConfig.cs [hunk 1]
```diff
@@ -1,34 +0,0 @@
-﻿using BenchmarkDotNet.Columns;
-using BenchmarkDotNet.Configs;
-using BenchmarkDotNet.Diagnosers;
-using BenchmarkDotNet.Engines;
-using BenchmarkDotNet.Jobs;
-using BenchmarkDotNet.Validators;
-
-namespace Microsoft.AspNetCore.SignalR.Microbenchmarks
-{
-    public class CoreConfig : ManualConfig
```
Comment (muratg) [inline benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/CoreConfig.cs]: Is this configuration no longer required, or did the logic move somewhere else?

Comment (pakrym) [reply benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/CoreConfig.cs]: Moved to common: https://github.com/aspnet/Common/pull/303

**Comment by pakrym** [comment]
Travis failure is due to unrelated flakiness

File: benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1 @@
+[assembly: BenchmarkDotNet.Attributes.AspNetCoreBenchmark]
```
Comment (natemcmaster) [inline benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/AssemblyInfo.cs]: You don't need to change this...i'm just curious. Would does this work if you put it into the csproj file?
```xml
<ItemGroup>
   <AssemblyAttribute Include="BenchmarkDotNet.Attributes.AspNetCoreBenchmark" />
</ItemGroup>
```
If so, you could add this piece of MSBuild into the Benchmark.Sources package so it's automatically applied to any benchmark project that uses it.

Comment (pakrym) [reply benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/AssemblyInfo.cs]: I don't want things that implicit. There might be a case where configs need to be overridden and having source file is much easier than trying to find some packages target that brought attribute and figure out how to change it.

Comment (pakrym) [reply benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/AssemblyInfo.cs]: I could've just dropped assembly info into source package too.

**Pull request merged**